types/index.d.ts

```diff
@@ -0,0 +1,328 @@
+        stopRecord?: string;
+        cancelRecord?: string;
+        retry?: string;
+        pleaseAllow?: {
+          title?: string;
+          text?: string;
+        };
+        notFound?: {
+          title?: string;
+          text?: string;
+        }
+      };
+      preview?: {
+        unknownName?: string;
+        change?: string;
+        back?: string;
+        done?: string;
+        unknown?: {
+          title?: string;
+          done?: string;
+        };
+        regular?: {
+          title?: string;
+          line1?: string;
+          line2?: string;
+        };
+        image?: {
+          title?: string;
+          change?: string;
+        };
+        crop?: {
+          title?: string;
+          done?: string;
+          free?: string;
+        };
+        video?: {
+          title?: string;
+          change?: string;
+        };
+        error?: {
+          default?: {
+            title?: string;
+            text?: string;
+            back?: string;
+          };
+          image?: {
+            title?: string;
+            text?: string;
+            back?: string;
+          };
+          size?: {
+            title?: string;
+            text?: string;
+          };
+          loadImage?: {
+            title?: string;
+            text?: string;
+          }
+        };
+        multiple?: {
+          title?: string;
+          question?: string;
+          tooManyFiles?: string;
+          tooFewFiles?: string;
+          clear?: string;
+          done?: string;
+          file?: {
+            preview?: string;
+            remove?: string;
+          }
+        }
+      }
+    };
+    footer?: {
+      text?: string;
+      link?: string;
+    }
+  };
+}
+
+export interface GeoLocation {
+  latitude: number;
+  longitude: number;
+}
+
+export interface OriginalImageInfo {
+  height: number;
+  width: number;
+  geo_location: null | GeoLocation;
+  datetime_original: string;
+  format: string;
+  color_mode: string;
+  dpi: null | number[];
+  orientation: null | number;
+  sequence?: boolean;
+}
+
+export type Uuid = string;
+
+export interface SourceInfo {
+  source: string;
+  file: any;
+}
+
+export interface FileInfo {
+  uuid: Uuid;
+  name: null | string;
+  size: null | number;
+  isStored: null | boolean;
+  isImage: null | boolean;
+  originalImageInfo: null | OriginalImageInfo;
+  mimeType: string;
+  originalUrl: null | string;
+  cdnUrl: null | string;
+  cdnUrlModifiers: null | string;
+  sourceInfo: null | SourceInfo;
+}
+
+export type OnTabVisibilityCallback = (tab: string, shown: boolean) => void;
+
+export interface DialogApi {
+  addFiles(files: FileInfo[]): void;
+  switchTab(tab: string): void;
+  fileColl: FileInfo[];
+  hideTab(tab: string): void;
+  showTab(tab: string): void;
+  isTabVisible(tab: string): boolean;
+  onTabVisibility(callback: OnTabVisibilityCallback): void;
+}
+
+export interface Settings {
+  // developer hooks
+  live?: boolean;
+  manualStart?: boolean;
```

Comment:
last thing ^-^
Please, remove this props, because I set they manually. 
If user provide `live` prop to widget component, it will be fail

Reply:
@jeetiss remove `live` and `manualStart`?